.github/dependabot.yml

```diff
@@ -9,3 +9,7 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "weekly"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
```

Comment:
Lines 14-15 are duplicative of lines 10-11?

Should `weekly` be `"weekly"` with quotes?

Reply:
Good question. I copied and pasted from PFE, as I’m on my phone. 

https://github.com/zooniverse/Panoptes-Front-End/blob/master/.github/dependabot.yml

Reply:
I don’t think the quotes matter.